### PR TITLE
Manifests and pipeline for rest proxy

### PR DESCRIPTION
**Pull request by martimors**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+version: "3.5"
+services:
+  schema-registry:
+    image: confluentinc/cp-schema-registry:${IMAGE_TAG}
+    hostname: schema-registry
+    container_name: schema-registry
+    env_file: .env
+    depends_on:
```
Comment (dfdsmmari) [inline docker-compose.yaml:8]: There is no broker?

File: k8s/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  selector:
+    matchLabels:
+      app: datadelivery-kafka-rest-proxy
+  template:
+    metadata:
+      labels:
+        app: datadelivery-kafka-rest-proxy
+    spec:
+      containers:
+        - image: confluentinc/cp-kafka-rest:6.2.0 # TODO: Paremetrize tag
```
Comment (dfdsmmari) [inline k8s/deployment.yaml]: Couldn't you just parametrize the tag and add it to the variable groups?

**Review by dfdsmmari — APPROVED** [review]

File: docker-compose.yaml (hunk 1 above)
Comment (martimors) [reply docker-compose.yaml:8]: No, I tested it with the dev kafka cluster

File: k8s/deployment.yaml (hunk 2 above)
Comment (martimors) [reply k8s/deployment.yaml]: I could. That's a remnant from the schema registry deployment so might as well get that done now...

File: docker-compose.yaml (hunk 1 above)
Comment (dfdsmmari) [reply docker-compose.yaml:8]: And it still works with depends_on? In that case I guess it is okay to leave it?

**Pull request merged**